5.5/run-mysqld.sh

```diff
@@ -94,6 +103,29 @@ if [ "$1" = "mysqld" -a ! -d "$MYSQL_DATADIR/mysql" ]; then
 		EOSQL
 	fi
 	mysqladmin $admin_flags flush-privileges shutdown
+}
+
+function generate_config() {
+	# Time zone.
```

Comment:
don't do this, check mysql docs for default TZ and set that, without this if/else. Also don't store configuration option in variables, only values...